syncano/models/channels.py

```diff
@@ -138,7 +138,7 @@ class Meta:
 
     def poll(self, room=None, last_id=None, callback=None, error=None, timeout=None):
         properties = self.get_endpoint_data()
-        endpoint = self._meta.resolve_endpoint('poll', properties)
+        endpoint = self._meta.resolve_endpoint('poll', properties, 'GET')
```

Comment:
:P you're using in all previous one the pattern:
http_method = 'GET'

self._meta.resolve(..., http_method)

maybe just add:
`endpoint = self._meta.resolve_endpoint('poll', properties, http_method='GET')`

or make http_method='GET' as default